src/test/forkscsv_tests.cpp

```diff
@@ -40,8 +42,11 @@ BOOST_AUTO_TEST_CASE(forkscsv_read_test)
 
 /**
  * Test CSV dump of built-in defaults.
+ * NOTE: the dump of 'main' data in this test depends on forkscsv_read_test
+ * as long as the 'segwit' line is not built-in.
+ * That is why an explicit dependency has been declared here.
  */
-BOOST_AUTO_TEST_CASE(forkscsv_dumpforks_test)
+BOOST_AUTO_TEST_CASE(forkscsv_dumpforks_test, * utf::depends_on("forkscsv_tests/forkscsv_read_test"))
 {
```

Comment:
I'm getting a compile error here "Boost auto testcase passed 2 args but only 1 expected"...I"m wondering what version of boost you're running when you compile?  I"m on 1.58

Reply:
I commented out the utf::depends_on and the tests run just fine...not sure why you need that or if it's really need it maybe you can just combine the two tests into one?